### ISSUE TITLE
Application hangs when searching summoner by name

### ISSUE THREAD

**Issue by SeaRoth** [andi2garcia/OnLoL-Riot-API#15]
# Description: 

- Searching a summoner results in broken app

# Steps to reproduce:
1. Download a fresh copy of OnLol-Riot-API
2. Enter your API key
3. Change the API call in index.php to $api->summonerByName("dyrus", "na");
4. Wait

# Result:
1. Applications hangs for a long period of time and a result never comes 

# Expected Result:
1. Return information regarding Dyrus 

# Other notes:
1. Making other API calls work (the default ones in the index.php)
**Error on PC:**
![image](https://cloud.githubusercontent.com/assets/3461713/25774652/acf288e2-3248-11e7-904b-d26557192a77.png)
**Error on MAC:**
![snip20170506_1](https://cloud.githubusercontent.com/assets/3461713/25774727/1b4a32a8-324a-11e7-9318-cffac9474249.png)

**Comment by SeaRoth** [comment]
I have reproduced the issue on my MAC using a fresh copy of OnLol-Riot-API-master

**Comment by SeaRoth** [comment]
![image](https://cloud.githubusercontent.com/assets/3461713/25774920/a68285a2-324d-11e7-84d9-fbca66cd9006.png)

**The API call being made:**
https://NA1.api.pvp.net/api/lol/NA1/v3/summoners/by-name/dyrus?api_key=xxx

**should read:**
https://na1.api.riotgames.com/lol/summoner/v3/summoners/by-name/dyrus?api_key=xxx

**Comment by d3sd1** [comment]
Yeah that's all because I pulled another copy of the source. Check now if pos downloading new code.

**Comment by SeaRoth** [comment]
What do you mean check now because OnLoL-Riot-API hasn't been updated for five days which means if I download it again then I will get the same error 

![image](https://cloud.githubusercontent.com/assets/3461713/25777105/bc3564a2-3287-11e7-8429-0a7ecc18a6e9.png)

**Comment by SeaRoth** [comment]
Maybe you meant that you're checking now, silly language barrier xD

**Comment by SeaRoth** [comment]
![image](https://cloud.githubusercontent.com/assets/3461713/25812783/373ebb36-33cc-11e7-8339-913b4581193f.png)

**Comment by SeaRoth** [comment]
I have taken it upon myself to fix the code.

I have gotten it to work - I just need to update all the other functions.

Here is a picture of how I got the summonerByName to work:

![image](https://cloud.githubusercontent.com/assets/3461713/25814194/71a5b9b4-33d1-11e7-94b2-dccfd8b110e0.png)


![image](https://cloud.githubusercontent.com/assets/3461713/25814145/3bfeb9f0-33d1-11e7-829a-7d32403a3a73.png)

![image](https://cloud.githubusercontent.com/assets/3461713/25814164/521f9592-33d1-11e7-9945-23937206e998.png)
